generic/OpenMP_Support.F90

```diff
@@ -519,20 +523,22 @@ function make_subgridcomps(GridComp, run_entry_points, num_grids, unusable, rc)
         character(len=ESMF_MAXSTR) :: comp_name
         character(len=:), allocatable :: labels(:)
         integer :: phase
+        type(ESMF_Config) :: CF
 
         allocate(subgridcomps(num_grids))
 
         call ESMF_VMGetCurrent(vm, _RC)
         call ESMF_VMGet(vm, localPET=myPET, _RC)
 
-        call ESMF_GridCompGet(GridComp, name=comp_name, _RC)
+        call ESMF_GridCompGet(GridComp, name=comp_name, CONFIG=CF, _RC)
```

Comment:
CONFIG should not be all caps

Reply:
`CONFIG` has been removed because it is not needed.